src/interaction/interactive_api.jl

```diff
@@ -143,16 +143,16 @@ function select_rectangle(scene; kwargs...)
     rect_ret = Node(FRect(0, 0, 1, 1)) # returned rectangle
 
     # Create an initially hidden rectangle
-    rect_vis = lines!(
-        scene, rect, raw = true, visible = false, kwargs...,
+    plotted_rect = lines!(
+        scene, rect, raw = true, visible = false, color = RGBAf0(0.1, 0.1, 0.8, 0.5), kwargs...,
```

Comment:
Let's make this themable!

Reply:
I don't understand what you are saying, please be explicit.

Reply:
I just mean that it should be passable as a kwarg, so that the color isn't hardcoded.

Reply:
The color isn't hardcoded, this is not how keywords work. If there is a keyword `color` in the propagated kwargs, it overwrites the "hardcoded".

Reply:
Huh, didn't know about that.  Pretty neat!